src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field block">
+        <legend>Personal information</legend>
+        <div class="input">
+          Surname:
+          <input type="text" 
+          name="attribute" 
+          autocomplete="off" 
+          required
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="attribute" 
            autocomplete="off" 
            required
          >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/220852632-7fe0848c-320d-49e8-9b0b-3865bb0a42b7.png)